src/_pytest/assertion/util.py

```diff
@@ -168,29 +171,7 @@ def assertrepr_compare(config, op: str, left: Any, right: Any) -> Optional[List[
     return [summary] + explanation
 
 
-def _compare_eq_any(left: Any, right: Any, verbose: int = 0) -> List[str]:
```

Comment:
Just a note, moving code around makes it harder to review, because it breaks the diff view, so best to do it separately. 

I guess this is a relevant comment for this PR :smile: